<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>colorDialog</class>
 <widget class="QDialog" name="colorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>700</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>850</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>850</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Color Selector Pallet List and Converter</string>
  </property>
  <property name="windowIcon">
   <iconset resource="testResource1.qrc">
    <normaloff>:/new/prefixTest/BGlogo</normaloff>:/new/prefixTest/BGlogo</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">colorDialog {
background-color: qlineargradient(spread:repeat, x1:0, y1:0, x2:0.955056, y2:1, stop:0 rgba(136, 136, 136, 255), stop:0.494382 rgba(142, 161, 179, 255), stop:0.994382 rgba(155, 127, 106, 255));
}
</string>
  </property>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>80</y>
     <width>69</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(157, 143, 126);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    color: yellow;
}
QPushButton:hover{
    border-color: rgb(170, 255, 127);
    background-color: rgb(200, 182, 161);
    color: black;
}
QPushButton:pressed {
    background-color: rgb(136, 124, 109);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Add -&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>20</y>
     <width>211</width>
     <height>151</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    min-width: 5em;
}
QPushButton:hover{
    border-color: rgb(170, 255, 127);
}
QPushButton:pressed {

    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Select Color</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>180</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RGB 0-255</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="RGB255Red">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>210</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: red;
    background-color: rgb(255, 165, 162);
    font: bold 14px;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="RGB255Gren">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>250</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: green;
    background-color: rgb(130, 192, 130);
    font: bold 14px;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="RGB255Blue">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>290</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: blue;
	background-color: rgb(181, 181, 255);
   font: bold 14px;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="RGB0_1Gren">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>250</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: green;
	background-color: rgb(130, 192, 130);
   font: bold 14px;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="RGB0_1Blue">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: blue;
	background-color: rgb(181, 181, 255);
   font: bold 14px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>180</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>RGB 0-1 </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="RGB0_1Red">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: red;
	background-color: rgb(255, 165, 162);
   font: bold 14px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>217</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>295</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>258</y>
     <width>47</width>
     <height>14</height>
    </rect>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="CMYK_Cyan">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>370</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: blue;
	background-color: rgb(0, 255, 255);
   font: bold 14px;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="CMYK_Meg">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>410</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: red;
	background-color: rgb(255, 0, 255);
   font: bold 14px;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="CMYK_Yelow">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>450</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: green;
    background-color: yellow;
    font: bold 14px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>340</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>CMYK 0-255</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="CMYK_Black">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: black;
    font: bold 14px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>374</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Cyan </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>415</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Magenta</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>456</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yellow</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>495</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="CMYK_MegF">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>410</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: red;
	background-color: rgb(255, 0, 255);
   font: bold 14px;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="CMYK_CyanF">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>370</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: blue;
	background-color: rgb(0, 255, 255);
    font: bold 14px;
}</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="CMYK_YelowF">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>450</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: green;
    background-color: yellow;
    font: bold 14px;
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>340</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;CMYK 0-1&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="CMYK_BlackF">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>490</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: black;
    font: bold 14px;
}</string>
   </property>
  </widget>
  <widget class="QTreeWidget" name="treeWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>30</y>
     <width>451</width>
     <height>491</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTreeWidget {
	border-style: inset;
	border-color: rgb(0, 0, 0);
	border-width: 3px;
	font: bold 20px;
}

QTreeWidget::branch:has-siblings:!adjoins-item {
    border-image: url(:/new/prefixTest/VLine) 0;
}

QTreeWidget::branch:has-siblings:adjoins-item {
	border-image: url(:/new/prefixTest/branch-more)0;
}

QTreeWidget::branch:!has-children:!has-siblings:adjoins-item {
    border-image: url(:/new/prefixTest/branch-end) 0;
}

QTreeWidget::branch:has-children:!has-siblings:closed,
QTreeWidget::branch:closed:has-children:has-siblings {
        border-image: none;
        image: url(:/new/prefixTest/branch-closed);
}

QTreeWidget::branch:open:has-children:!has-siblings,
QTreeWidget::branch:open:has-children:has-siblings  {
        border-image: none;
        image: url(:/new/prefixTest/branch-open);
}</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="sizeAdjustPolicy">
    <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectItems</enum>
   </property>
   <attribute name="headerDefaultSectionSize">
    <number>150</number>
   </attribute>
   <column>
    <property name="text">
     <string notr="true">1</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>530</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(157, 143, 126);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    min-width: 5em;
    color: yellow;
}
QPushButton:hover{
    border-color: rgb(170, 255, 127);
    background-color: rgb(200, 182, 161);
    color: black;
}
QPushButton:pressed {
    background-color: rgb(136, 124, 109);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Add New Object Model</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>530</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(157, 143, 126);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    min-width: 5em;
    color: yellow;
}
QPushButton:hover{
    border-color: rgb(170, 255, 127);
    background-color: rgb(200, 182, 161);
     color: black;
}
QPushButton:pressed {
    background-color: rgb(136, 124, 109);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Add New Object Item or Item Color</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_5">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>590</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(157, 143, 126);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    min-width: 5em;
    color: yellow;
}
QPushButton:hover{
	border-color: rgb(170, 255, 127);
	background-color: rgb(200, 182, 161); 
   color: black;
}
QPushButton:pressed {
	background-color: rgb(136, 124, 109);
    border-style: inset;
}
</string>
   </property>
   <property name="text">
    <string>Load Pallet</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_6">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>590</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(157, 143, 126);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    min-width: 5em;
    color: yellow;
}
QPushButton:hover{
    border-color: rgb(170, 255, 127);
    background-color: rgb(200, 182, 161); 
    color: black;
}
QPushButton:pressed {
    background-color: rgb(136, 124, 109);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Save Pallet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_7">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(157, 143, 126);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    min-width: 5em;
    color: yellow;
}
QPushButton:hover{
    border-color: rgb(170, 255, 127);
    background-color: rgb(200, 182, 161); 
    color: black;
}
QPushButton:pressed {
    background-color: rgb(136, 124, 109);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Create New Pallet</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_8">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>640</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
	background-color: rgb(157, 143, 126);
    border-style: outset;
    border-width: 2px;
    border-radius: 10px;
    border-color: black;
    min-width: 5em;
    color: yellow;
}
QPushButton:hover{
    border-color: rgb(170, 255, 127);
    background-color: rgb(200, 182, 161); 
    color: black;
}
QPushButton:pressed {
    background-color: rgb(136, 124, 109);
    border-style: inset;
}</string>
   </property>
   <property name="text">
    <string>Save as Defult</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_9">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>600</y>
     <width>172</width>
     <height>86</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::NoContextMenu</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
 border-style: none;
background-image: url(:/new/prefixTest/DGLogo);
}</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
   <property name="text">
    <string notr="true"/>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>530</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>HTML</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="HTML_Color">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>530</y>
     <width>110</width>
     <height>31</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">QTextEdit {
   border-style: inset;
    border-width: 2px;
    border-radius: 2px;
    border-color: black;
    font: bold 14px;
}</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="testResource1.qrc"/>
 </resources>
 <connections/>
</ui>
